README.md

```diff
@@ -1,2 +1,132 @@
-# final-capstone-frontend
-Final capstone frontend
+# EASY BOOKING APP
```

Comment:
- [ ] Your project does not have a license.  Kindly add it and link to it on your readme in a new section called "License".

 ## Tips and extra help

Go [here] (https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in a new "License" section of your readme.

That way, your app will be well documented 💪